implementation/choreography-to-orchestration/lib/orchestration-stack.ts

```diff
@@ -0,0 +1,264 @@
+import { Construct } from "constructs";
+import {
+  Stack,
+  StackProps,
+  aws_lambda as lambda,
+  aws_apigateway as apigateway,
+  aws_iam as iam,
+  aws_dynamodb as dynamodb,
+  aws_sns as sns,
+  aws_sqs as sqs,
+  aws_lambda_destinations as destinations,
+  aws_lambda_event_sources as sources,
+  aws_stepfunctions as sfn,
+  aws_stepfunctions_tasks as tasks,
+  CfnOutput,
+} from "aws-cdk-lib";
+import { NagSuppressions } from 'cdk-nag'
+
+
+export class OrchestrationStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    NagSuppressions.addStackSuppressions(this, [
+      {
+        id: 'AwsSolutions-IAM4',
+        reason: 'This is a demo application, allowing use of AWS Managed policies'
+      },
+      { 
+        id: 'AwsSolutions-SF1', 
+        reason: 'This is demo stack, hence not enabling Cloudwatch logging from StepFunction' 
+      },
+      { 
+        id: 'AwsSolutions-SF2', 
+        reason: 'This is demo stack, hence not enabling X-Ray tracing' 
+      },
+    ]);
+
+    const ProcessPaymentFunction = new lambda.Function(this, "ProcessPaymentFunction", {
+      description: "",
+      functionName: "ProcessPaymentFunction",
+      handler: "ProcessPayment.lambda_handler",
+      code: lambda.Code.fromAsset("lambda/orchestration"),
+      memorySize: 128,
+      runtime: lambda.Runtime.PYTHON_3_9,
```

Comment:
Lambda here is using 3.9 instead of 3.10. This throws cdk-nag error "AwsSolutions-L1"